examples/extensions-zalando/src/main/java/org/zalando/intellij/swagger/examples/extensions/zalando/validator/zally/ZallyYamlFileValidator.java

```diff
@@ -11,50 +11,33 @@
 import org.zalando.intellij.swagger.examples.extensions.zalando.validator.ZallySettings;
 import org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.model.LintingResponse;
 import org.zalando.intellij.swagger.examples.extensions.zalando.validator.zally.model.Violation;
-import org.zalando.intellij.swagger.file.FileDetector;
-import org.zalando.intellij.swagger.traversal.path.PathFinder;
 
 import java.util.List;
 import java.util.Optional;
 
-public class YamlFileValidator extends LocalInspectionTool {
+public abstract class ZallyYamlFileValidator extends LocalInspectionTool {
 
-    private final static Logger LOG = Logger.getInstance(YamlFileValidator.class);
+    private final static Logger LOG = Logger.getInstance(OpenApiYamlFileValidator.class);
```

Comment:
Do you really want to log using `OpenApiYamlFileValidator` (a child class of this abstract class)?

Reply:
Certainly not, fixed.